odl/trafos/backends/pywt_bindings.py

```diff
@@ -275,6 +338,25 @@ def pywt_coeffs_from_flat_array(arr, shapes):
         ``Di`` the tuple of i-th level detail coefficient arrays. Each of
         the ``Di`` tuples has length ``2 ** ndim - 1``, where ``ndim`` is
         the number of dimensions of ``arr``.
+
+    Examples
+    --------
+    Turn the flat array from the example in `pywt_flat_array_from_coeffs`
+    back into a coefficient list:
+
+    >>> arr = [1, 1, 2, 2, 3, 3, 4, 4]
+    >>> # approximation and detail coefficient shapes
+    >>> shapes = [(1, 2), (1, 2)]
+    >>>
+    >>> coeffs = pywt_coeffs_from_flat_array(arr, shapes)
+    >>> approx, details = coeffs
+    >>> print(approx)
+    [[1 1]]
+    >>> for detail in details:
```

Comment:
I'll have to agree, but I simply felt that it is easier to maintain and this is a internal function anyway so being complicated isn't that bad